corehq/apps/registration/templates/registration/base.html

```diff
@@ -2,6 +2,23 @@
 
 {% block head %}{{ block.super }}
     {% include 'analytics/fullstory.html' %}
+    {% if FACEBOOK_PIXEL_ID %}
```

Comment:
this needs to be loaded into the template context right?

Reply:
there is middleware that handles this. I tested it locally

Reply:
middleware takes care of it